docs/source/reader_and_writer.rst

```diff
@@ -135,6 +135,106 @@ Basically, it's just a high level api on top of the native
     df.write_ext.mysql('localhost', 'my_database', 'my_table',
                        options={'user': 'root', 'password': 'root'})
 
+.. _kafka:
+
+Kafka
+^^^^^
+
+Spark streaming Kafka package (`org.apache.spark:spark-streaming-kafka <https://mvnrepository.com/artifact/org.apache.spark/spark-streaming-kafka-0-10_2.11/2.0.0>`_)
```

Comment:
Sparkly's reader and writer for Kafka are built on top of the official spark package for Kafka and python library kafka-python. The first one allows us to read data efficiently, the second covers a lack of writing functionality in the official distribution.